tools/tasks/tslint.ts

```diff
@@ -11,7 +11,9 @@ export = function tslint(gulp, plugins) {
     ];
 
     return gulp.src(src)
-      .pipe(plugins.tslint())
+      .pipe(plugins.tslint({
+        rulesDirectory: NG2LINT_RULES
```

Comment:
Maybe that one should be included in `tsconfig` instead to leverage linting straight from the IDE.